### PR TITLE
Level 2: Add level 2

### PR DESCRIPTION
**Pull request by LordMittens**

### CODE REVIEW

**Review by Westlanderz — APPROVED** [review]
LGTM

File: Assets/Scripts/Core/Controllers/GameController.cs [hunk 1]
```diff
@@ -69,8 +69,11 @@ public class LevelTransitionArgs : System.EventArgs {
 
 			if(SceneManager.GetActiveScene().name == "TutorialLevel") {
 				LoadTilemapData("tutorial_V4.5");
+			} else if(SceneManager.GetActiveScene().name == "PrisonLevel") {
+				// LoadTilemapData("prisonlevel");
 			} else if(SceneManager.GetActiveScene().name == "Final Level") {
 				LoadTilemapData("finallevel_V1.6");
+
```
Comment (Dennisbonke) [inline Assets/Scripts/Core/Controllers/GameController.cs:76]: Why is this empty line here?

**Review by Dennisbonke — CHANGES_REQUESTED** [review]
One small nitpick

**Review by Dennisbonke — APPROVED** [review]
LGTM

**Pull request merged**